service/start.go

```diff
@@ -29,14 +29,22 @@ func (s *Service) Start(c container.Container) (serviceIDs []string, err error)
 	}
 	// BUG: https://github.com/mesg-foundation/core/issues/382
 	// After solving this by docker, switch back to deploy in parallel
-	serviceIDs = make([]string, len(s.Dependencies))
-	for i, dep := range s.Dependencies {
+	serviceIDs = make([]string, 0)
+	for _, dep := range s.Dependencies {
 		serviceID, err := dep.Start(c, s, networkID)
```

Comment:
same pattern https://github.com/mesg-foundation/core/pull/860/files#diff-be8c3a41ba62becfd4d278926d06a683R27 can be applied here by having a `start := func()` above. but not blocking for this pr.

Reply:
done